src/modules/airspeed_selector/airspeed_selector_main.cpp

```diff
@@ -398,10 +398,11 @@ AirspeedModule::Run()
 
 			}
 
-			// save estimated airspeed scale after disarm
+			// save estimated airspeed scale after disarm if airspeed is valid and scale has changed by more then 1%
```

Comment:
@sfuhrer This comment I think is not quite correct, since you are not really checking for a percentage, are you?
Maybe would be a good time to also make a constant class member and document it there.

Reply:
yeah we do..
`fabsf(_airspeed_validator[i].get_CAS_scale_validated() - _param_airspeed_scale[i]) > 0.01f`
Yes can make it cleaner!

Reply:
Actually I challenge the existence of this random 1% cliff for updating the parameter. Removed in my last commit. Do you agree? I don't see what we loose by not updating the param with the highest precision we have.